src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,6 +9,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_NAME = new Prefix("-n");
     public static final Prefix PREFIX_PHONE = new Prefix("-p");
     public static final Prefix PREFIX_EMAIL = new Prefix("-e");
+    public static final Prefix PREFIX_STARTDATE = new Prefix("-s");
+    public static final Prefix PREFIX_DEADLINE = new Prefix("-d");
```

Comment:
I think perhaps this could be changed to PREFIX_ENDDATE for better readability and maybe the prefix could be -sd and -ed